geopackage_validator/validate.py

```diff
@@ -40,7 +39,7 @@ def get_all_validations():
     used_validations = list(
```

Comment:
Dit is geen review op jouw code. Maar iets wat ik tijdens mijn eerste review over het hoofd had gezien. Zou je voor mij deze nog willen aanpassen naar: 
```
used_validations = [v["validation_code"] for k, v in VALIDATIONS.items() if v["validation_code"].startswith("R")]
```

`filter` is vaak niet nodig en het is gebruikelijker een listcomprehension te gebruiken. Daarnaast wil je lambdafuncties  graag vermijden.

Reply:
Overigens zijn er natuurlijk wel situaties waarbij filter en lambda functies relevant zijn.